src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -159,35 +163,27 @@ public void serializeToView(ByteBuf buf)
         buf.writeInt(currentLevel);
         buf.writeInt(levels.size());
 
-        for(Level level : levels)
+        for (Level level : levels)
         {
             buf.writeInt(level.getNumberOfNodes());
         }
     }
 
     /**
-     * Returns the depth limit
-     * Limitted by building level
-     *      - Level 1: 50
-     *      - Level 2: 30
-     *      - Level 3: 5
+     * Returns the depth limit Limitted by building level - Level 1: 50 - Level
+     * 2: 30 - Level 3: 5
      *
-     * @return              Depth limit
+     * @return Depth limit
      */
     public int getDepthLimit()
     {
-        if(this.getBuildingLevel() == 1)
+        if (this.getBuildingLevel() == 1)
         {
             return 50;
-        }
-        else if(this.getBuildingLevel() == 2)
+        } else if (this.getBuildingLevel() == 2)
         {
             return 30;
-        }
-        else if(this.getBuildingLevel() == 3)
-        {
-            return 5;
-        }
+        } else if (this.getBuildingLevel() == 3) { return 5; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)